ci/pipeline.yml

```diff
@@ -1740,6 +1740,7 @@ jobs:
               NODES: 6
               FLAKE_ATTEMPTS: 2
               CONFIG_FILE_PATH: gyro-exp/baras_integration_config.json
+              LABEL_FILTER: "!v2_api"
```

Comment:
The exclamation mark could provoke a "event not found" error in the shell. In that case, we can turn of history substitution with `+H`.